recipes/screeninfo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "screeninfo" %}
+{% set version = "0.6.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/screeninfo-{{ version }}.tar.gz
+  sha256: 1c4bac1ca329da3f68cbc4d2fbc92256aa9bb8ff8583ee3e14f91f0a7baa69cb
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+  run:
+    - cython  # [osx]
```

Comment:
Why is cython required at *run* time? I see it's listed in `setup.py`, but there are no `.pyx` files in the package.

Reply:
It's required because of the OSX AppKit Internal Dependency.

Reply:
But you're taking care of that by depending on `pyobjc-framework-cocoa`. The `osx` enumerator in `screeninfo` just imports `AppKit`, which is provided by `pyobjc-framework-cocoa`. I don't see any line of code in `screeninfo` that directly uses cython.

Reply:
I have dug a little deeper, and actually, CPython is marked as a dependency in pip, but in reality, it's not required as so I have removed CPython ad pip check to stop the build from failing in OSX env. Hope that resolves your query.